test/test_helper.exs

```diff
@@ -1,2 +1,5 @@
+Mox.defmock(WoWData.Api.ClientBehaviourMock, for: WoWData.Api.ClientBehaviour)
```

Comment:
Roda um `mix format`

Reply:
Ele não fez nada, mas ja que é meio que uma config eu deixei assim:

`Mox.defmock
  WoWData.Api.ClientBehaviourMock,
  for: WoWData.Api.ClientBehaviour

Mox.defmock
  WoWData.Api.AccessTokenBehaviourMock,
  for: WoWData.Api.AccessTokenBehaviour
`